include/aspect/postprocess/visualization.h

```diff
@@ -462,6 +462,19 @@ namespace aspect
          */
         bool filter_output;
 
+        /**
+         * deal.II offers the possibility to write vtu files with higher order
```

Comment:
Capital D in Deal.II?

Reply:
Nope, it is usually deal.II, see https://dealii.org/. For reasons you would need to ask Wolfgang :smile:. It looks a bit funny at the beginning of a sentence, but I think it is consistent with how it is usually used.

Reply:
Since you where so consistent with it, I already suspected something like that. I just wanted to make sure :)